tests/tls_challenge_test.go

```diff
@@ -117,6 +119,8 @@ func (ts *TLSChallengeSuite) setupProxy(centralEndpoint string) {
 	ts.createProxyConfigMap(centralEndpoint, nginxConfigName)
 	ts.createService(ts.ctx, proxyNs, name, nginxLabels, map[int32]int32{443: 8443})
 	ts.createProxyDeployment(name, nginxLabels, nginxConfigName, nginxTLSSecretName)
+	ts.logf("Waiting for nginx proxy deployment to become ready...")
```

Comment:
Bonus points for moving this message into `waitUntilK8sDeploymentReady` as well if you'd be so inclined 😉